integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java

```diff
@@ -823,4 +830,79 @@ public static void verifyPodIntrospectVersionUpdated(Set<String> podNames,
                   podIntrospectVersionUpdated(podName, domainNamespace, expectedIntrospectVersion));
     }
   }
+
+  /**
+   * Change the WebLogic Admin credential of the domain.
+   * Patch the domain CRD with a new credentials secret.
+   * Update domainRestartVersion to trigger a rolling restart of server pods.
+   * Make sure all the server pods are re-started in a rolling fashion.
+   * Check the validity of new credentials by accessing WebLogic RESTful Service.
+   * @param domainNamespace namespace where the domain is
+   * @param domainUid domain uid for which WebLogic Admin credential is being changed
+   * @param adminServerPodName pod name of admin server
+   * @param managedServerPrefix prefix of the managed server
+   * @param replicaCount replica count of the domain
+   * @param args arguments to determine appending suffix to managed server pod name or not.
+   *             Append suffix if it's set. Otherwise do not append.
+   */
+  @DisplayName("Change the WebLogic Admin credential of the domain")
+  public static void verifyMiiUpdateWebLogicCredential(String domainNamespace, String domainUid,
```

Comment:
change the method name to updateAdminCredentialsAndVerify or some such ...

Reply:
Also does this method work only for MII domains? If not, it should go in CommonTestUtils

Reply:
Yes, this method works only for Mii domain. I follow the same name style in the CommomMiiTestUtils.